test/legislature_test.rb

```diff
@@ -78,4 +78,46 @@ def test_csv_url_method
       assert_equal legislature.names_url, URI.join(base, sha + path).to_s
     end
   end
+
+  def test_upper_house_method
+    VCR.use_cassette('countries_json') do
+      bicameral = Everypolitician.country(code: 'CM')
+      assert_equal 'Sénat', bicameral.upper_house.name
+    end
+  end
+
+  def test_upper_house_with_unicameral_house
+    VCR.use_cassette('countries_json') do
```

Comment:
rather than having every test have to go through the `use_cassette('countries_json')` dance, it might be worth using `minitest-around`

(See, for example, https://github.com/everypolitician/wikisnakker/blob/master/test/identifiers_test.rb)